src/ResourceManager/Profile/Commands.Profile.Test/ArgumentCompleterTests.cs

```diff
@@ -48,5 +48,19 @@ public void TestResourceGroupCompleter()
         {
             ProfileController.NewInstance.RunPsTest(xunitLogger, "72f988bf-86f1-41af-91ab-2d7cd011db47", "Test-ResourceGroupCompleter");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.Flaky)]
```

Comment:
Why did you add flaky?

Reply:
copied from other completes tests.
fixed